docs/types/conversion.rst

```diff
@@ -174,15 +174,15 @@ converted to any fixed-size bytes type:
     bytes4 g = 0x0; // fine
 
 String literals and hex string literals can be implicitly converted to fixed-size byte arrays,
-if their number of characters matches the size of the bytes type:
+if their number of characters is less than or equal to the size of the bytes type:
 
 .. code-block:: solidity
 
     bytes2 a = hex"1234"; // fine
     bytes2 b = "xy"; // fine
-    bytes2 c = hex"12"; // not allowed
+    bytes2 c = hex"12"; // fine
     bytes2 d = hex"123"; // not allowed
```

Comment:
```suggestion
```

I'd remove this completely, since this has nothing to do with conversions, and is already described in the section above.